src/pages/Autolending/AutolendingWhen.vue

```diff
@@ -0,0 +1,248 @@
+							v-model="donationToggle"
+						>
+							Include a donation to Kiva of
+							<kv-dropdown-rounded v-model="donation">
+								<option :value="0">
+									0%
+								</option>
+								<option :value="5">
+									5%
+								</option>
+								<option :value="10">
+									10%
+								</option>
+								<option :value="15">
+									15%
+								</option>
+								<option :value="20">
+									20%
+								</option>
+							</kv-dropdown-rounded>
+						</kv-radio>
+						<kv-radio
+							data-test="is-autolending-donation-off"
+							id="is-autolending-donation-off"
+							radio-value="off"
+							v-model="donationToggle"
+						>
+							No donation to Kiva
+						</kv-radio>
+					</div>
+					<template slot="controls">
+						<hr>
+						<kv-button
+							data-test="when-save-button"
+							class="smaller button"
+							v-if="!isSaving"
+							@click.native="save"
+							:disabled="!isChanged"
+						>
+							Save
+						</kv-button>
+						<kv-button data-test="when-save-button" class="smaller button" v-else>
+							Saving <kv-loading-spinner />
+						</kv-button>
+					</template>
+				</kv-lightbox>
+
+				<lend-timing-messaging />
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import gql from 'graphql-tag';
+import _isFinite from 'lodash/isFinite';
+import KvIcon from '@/components/Kv/KvIcon';
+import KvLightbox from '@/components/Kv/KvLightbox';
+import KvButton from '@/components/Kv/KvButton';
+import KvRadio from '@/components/Kv/KvRadio';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+import KvDropdownRounded from '@/components/Kv/KvDropdownRounded';
+
+import LendTimingDropdown from './LendTimingDropdown';
+import LendTimingMessaging from './LendTimingMessaging';
+
+
+export default {
+	inject: ['apollo'],
+	components: {
+		KvIcon,
+		KvLightbox,
+		KvButton,
+		KvRadio,
+		KvLoadingSpinner,
+		KvDropdownRounded,
+		LendTimingDropdown,
+		LendTimingMessaging,
+	},
+	data() {
+		return {
+			isSaving: false,
+			isEnabled: false,
+			showLightbox: false,
+			isChanged: false,
+			lendAfterDaysIdle: 0,
+			donation: 15,
+			donationToggle: 'on',
+		};
+	},
+	apollo: {
+		query: gql`{
+			autolending @client {
+				profileChanged
+				currentProfile {
+					isEnabled
+					donationPercentage
+					lendAfterDaysIdle
+				}
+			}
+		}`,
+		preFetch: true,
+		result({ data }) {
+			this.isEnabled = !!_get(data, 'autolending.currentProfile.isEnabled');
+			const donationPercentage = _get(data, 'autolending.currentProfile.donationPercentage');
+			this.lendAfterDaysIdle = _get(data, 'autolending.currentProfile.lendAfterDaysIdle');
+			this.isChanged = !!_get(data, 'autolending.profileChanged');
+			this.donation = _isFinite(donationPercentage) ? donationPercentage : 15;
+			this.donationToggle = this.donation !== 0 ? 'on' : 'off';
+			// After initial value is loaded, setup watch
+			this.$watch('donation', (donation, previousDonation) => {
```

Comment:
What's the affect of enabling these watcher in the ssr context? Seems like we could be setting up some type of reactive code that can never be executed? Does it just hang around in the node app instance memory?

Reply:
I think we usually activate watchers once we know we're in the client...

Reply:
Yeah thats fair, I was just trying to avoid the initial firing of the watcher on setting the initial value. I'll look into it